<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpDlg</class>
 <widget class="QDialog" name="HelpDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help Contents</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonPrevious">
       <property name="toolTip">
        <string>Previous Page</string>
       </property>
       <property name="text">
        <string>Previous Page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext">
       <property name="toolTip">
        <string>Next Page</string>
       </property>
       <property name="text">
        <string>Next Page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFirst">
       <property name="toolTip">
        <string>First Page</string>
       </property>
       <property name="text">
        <string>First Page</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textBrowser</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HelpDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HelpDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPrevious</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>backward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNext</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonFirst</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>home()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textBrowser</sender>
   <signal>backwardAvailable(bool)</signal>
   <receiver>buttonPrevious</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textBrowser</sender>
   <signal>forwardAvailable(bool)</signal>
   <receiver>buttonNext</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
